emails/templates/emails/wrapped_email.html

```diff
@@ -38,9 +42,16 @@
             <tr>
               <td width="100%" align="center" style="line-height: 150%;">
                 <p style="color: #363959; font-size: 13px; font-family: sans-serif; margin-top: 0; margin-bottom: 0;">
-                  This message was forwarded from <span style="font-family: sans-serif; font-weight: bolder; color: #20123a; text-decoration: none; font-size: 13px;">{{ display_email|safe }}</span> by <a href="{{ SITE_ORIGIN }}" target="_blank" style="font-family: sans-serif; color: #20123a; text-decoration: underline; font-weight: bolder; font-size: 13px;">Firefox Relay</a>.
+                  {% bold_violet_text display_email as bolded_email %}
+                  {% bold_violet_link SITE_ORIGIN 'Firefox Private Relay' as linked_origin %}
+                  {% ftlmsg 'forwarded-email-header-from' display_email=bolded_email linked_origin=linked_origin as email_wrap_forwarded_from %}
+                  {{ email_wrap_forwarded_from|safe }}
```

Comment:
Because we have to inline all the styles in emails, all the messages follow this general flow:

1. Use the new template tags to store strings with markup in them as variables
2. Pass those variables into `ftlmsg` messages and store the marked-up AND translated messages as variables 
3. Render the marked-up AND translated messages with `|safe` to preserve the markup